test/test_Times.cc

```diff
@@ -0,0 +1,71 @@
+#include "gtest/gtest.h"
+#include <assert.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <math.h>
+#include "../generic.h"
+#include "../SW_Sky.h"
+#include "../filefuncs.h"
+#include "../myMemory.h"
+#include "../SW_Defines.h"
+#include "../SW_Files.h"
+#include "../SW_Model.h"
+#include "../SW_Site.h"
+#include "../SW_SoilWater.h"
+#include "../SW_VegProd.h"
+#include "../SW_Site.h"
+#include "../SW_Flow_lib.h"
+#include "../Times.h"
+#include "sw_testhelpers.h"
+
+namespace{
+  // Test the 'Times.c' function 'interpolate_monthlyValues'
+  TEST(TimesTest, interpolateMonthlyValues){
+    // point to the structure that contains cloud coverage monthly values
+    SW_SKY SW_Sky;
+    SW_SKY *interpolate = &SW_Sky;
+    unsigned int i;
+    // set all monthlyValues all 10
```

Comment:
// set all monthly values __to__ 10?

Reply:
Yes, since the test relies on the values here, I need to know exactly what the values are in order to test properly. Setting them all to 10 ensures this. I will add this to the comment to be more clear.